_pages/speakers.md

```diff
@@ -19,4 +19,6 @@ When we have a need for a speaker or see a session that our members have request
 
```

Comment:
I suggest to delete this statement. It does reflect a kind of discrimination. Our reasoning of limited budget is legitimate, however stating that this will make us prefer speakers from nearby areas defies our pride that CB is about diversity and inclusivity. I mean you had such cases of speakers coming from Canada and US and you dealt with it perfectly on a case by case basis and w/o hurting the budget. Plus those speakers might open the door for a potential sponsor.